ishop/API/permissions.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework import permissions
+
+
+class IsAdminOrReadOnly(permissions.BasePermission):
+    """
+    Custom permission to only allow admins to edit.
+    Retrieve - all users
+    """
+
+    def has_object_permission(self, request, view, obj):
+        if request.method in permissions.SAFE_METHODS:
```

Comment:
not an error but I think all permissions will looks better if will be rewritten for one-line